Editors: add 'validationMessagePosition' description (#3986)
DevExpress/devextreme-documentation: api-reference/10 UI Components/Editor/1 Configuration/validationMessagePosition.md

---
---
##### shortDescription

> Specifies the position of a validation message relative to the component. A validation message is a message about the [validation rules](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/) that this component's value does not satisfy.

Reviewer: ```suggestion
Specifies the position of a validation message relative to the component. The validation message describes the [validation rules](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/) that this component's value does not satisfy.
```
